### DISCUSSION TITLE
OData Query

### DISCUSSION THREAD

**Discussion by 4damW** [nbrown02/FlowViz#126] [Q&A] [answered]
Hi guys,

I'm wanting to retrieve data from the last 120 days using the OData queries.

I'm trying to do this
![image](https://github.com/user-attachments/assets/21601f56-27e5-4dbc-a98a-cfb87e28f98b)

I get the following error and I can't figure out how to format the datetime properly.
![image](https://github.com/user-attachments/assets/98f57a9e-6d40-430b-a1af-9abaca40427a)

Anyone have any ideas?

Thanks
Adam

**Comment by nbrown02** [comment, marked as answer]
Hey there @4damW :)

You can do it like so for the last 12 months (tweak to be for days your end):
![image](https://github.com/user-attachments/assets/31cb298a-9672-4c4c-a626-8c7af3a08142)

Worth noting, if you publish to the Power BI service, this will in fact disable auto-refresh ability, as I found via this issue #42

**Reply by 4damW** [reply]
cheers :)
